Main.py

```diff
@@ -11,22 +11,27 @@
 # \todo: HERE, Delete variables we don't need anymore.
 # data1 = data2 = None # delete unneccesary variables.
 
+lag_time = \
```

Comment:
Not entirely happy that `run()` uses variables outside its scope, but this can be fixed later.